.gitignore

```diff
@@ -1,10 +1,3 @@
+build
 node_modules
-dist
-*.log
-subgraph.yaml
-docker-compose.yml
-config.json
-abis
-yarn.lock
-src/types
-schema.graphql
\ No newline at end of file
+*.swp
```

Comment:
why remove these?

Reply:
A bug was introduced on my end because subgraph.yaml & schema.graphql were ignored & I didn't notice them not being committed. These definitely need to be removed from the .gitignore.

I got a little overexcited & shouldn't have removed dist or *.log or some of the others though, you're right

Reply:
subgraph.yaml & schema.graphql should not be in .gitignore .
it is generated automatically by https://github.com/daostack/subgraph/blob/master/subgraph.handlebars.yaml and other scripts .
Please read Readme ...on how to use the env